examples/connect/server.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+// eslint-disable-next-line
+const tracing = require('./tracing')('example-connect-server');
+
+// Require in rest of modules
+const connect = require('connect');
+const axios = require('axios');
+
+// Setup express
+const app = connect();
+const PORT = 8080;
+
+app.use(function middleware1(req, res, next) {
+  next();
+});
+
+app.use((req, res, next) => {
+  next();
+});
+
+app.use('/run_test', async (req, res) => {
+  const result = await axios.get('https://raw.githubusercontent.com/open-telemetry/opentelemetry-js/main/package.json');
+  tracing.log('sending response');
+  res.end(`OK ${result.data.version}`);
+  if (tracing.connectInstrumentation.isEnabled()) {
```

Comment:
I can only guess that the reason you disable the instrumentation is to, on every second request, showcase how the traces will look without Connect instrumentation.
A nit: Perhaps a comment this, or have another endpoint do that.

Reply:
added